src/server/models/types.ts

```diff
@@ -45,41 +42,15 @@ export interface ScheduledProcessKeyDates extends JsonObject {
   unpublished: UnpublishedKeyDates;
 }
 
-export interface CurrentAnnualReview extends JsonObject {
-  reference: string;
-  eligibleListItems: number[];
-  keyDates: ScheduledProcessKeyDates;
-}
-
-export interface ListJsonData extends JsonObject {
-  users?: string[];
-  currentAnnualReview?: CurrentAnnualReview;
-}
+export type CurrentAnnualReview = ListItem;
```

Comment:
`CurrentAnnualReview` would be found on `List.jsonData.currentAnnualReview` - is assigning `ListItem` correct here?

Reply:
Good spot. That's my mistake 🤦🏿